sorting/topological_sort.cpp

```diff
@@ -0,0 +1,98 @@
+/**
+ * @file
+ * @brief Topological sort algorithm
+ *
+ * Implementation Details -
+ *      Topological sorting for Directed Acyclic Graph (DAG) is a linear ordering
+ *	of vertices such that for every directed edge uv, vertex u comes before v in the
+ *	ordering. Topological Sorting for a graph is not possible if the graph is not a DAG.
+ */
+#include <algorithm>
+#include <iostream>
+#define MAX 10
+int n, adj[MAX][MAX];

```

Comment:
why don't you use ```std::vector<std::vector<int>> adj``` for creating graph, it will give more flexibility to program instead of just having 10 nodes using ```#define MAX 10```

for help refer to files in PR  #932

Reply:
Also, please avoid using Global variables. Pass variables as function pointers.